bootstrap.php

```diff
@@ -19,6 +22,66 @@
 	}
 	return;
 }
+
+
+/*
+ * Initialize coming soon module via container
+ */
+$nfd_module_container = new Container();
+// Set plugin to container
+$nfd_module_container->set(
+	'plugin',
+	$nfd_module_container->service(
+		function() {
+			return new Plugin(
+				array(
+					'id'   => 'hostgator',
+					'file' => HOSTGATOR_PLUGIN_FILE,
+				)
+			);
+		}
+	)
+);
+
+// Set coming soon values
+$nfd_module_container->set(
+	'comingsoon',
+	array(
+		'option_name'         => 'mm_coming_soon',
+		'admin_screen_id'     => 'hostgator',
+		'admin_app_url'       => admin_url( 'admin.php?page=hostgator#/home' ),
+		'template_h1'         => 'A New WordPress Site',
+		'template_h2'         => 'Coming Soon!',
```

Comment:
The two strings above need to be translatable.

Reply:
Yes, added the `__`